src/rest_framework_reactive/consumers.py

```diff
@@ -6,12 +6,50 @@
 from channels.consumer import AsyncConsumer, SyncConsumer
 from channels.generic.websocket import JsonWebsocketConsumer
 
+from django.core.cache import cache
+
 from .models import Observer, Subscriber
 from .observer import QueryObserver
 from .protocol import *
 
 # Maximum number of cached observer executors.
 MAX_CACHED_EXECUTORS = 1024
+# Waiting observers cache name
+WAITING_OBSERVERS_CACHE_PREFIX = 'waiting_observer_'
+# Rate at which observer evaluation is conducted (in seconds)
+THROTTLE_RATE = 2
+
+
+def evaluated_cache_key(observer_id):
```

Comment:
Rename to `_evaluated_cache_key` to make it clearer that this is an utility function.